docs/usage.md

```diff
@@ -14,10 +14,10 @@ You can get a propositional title and navigation by setting the content for `hea
 
     <div class="header-proposition">
       <div class="content">
-        <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
+        <a href="/" id="proposition-name">Service Name</a>
+        <button href="#proposition-links" class="js-header-toggle menu" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
```

Comment:
href is not valid `<button>` attribute. we either go back to `<a>` tag or remove that from the button.

Reply:
Unfortunately it would make the JS code non-backward compatible to remove it. Instead I've switched back a to a link but changed it's role to button to make it less confusing with VoiceOver.

Reply:
So back to a link.